working-with-your-data/working-with-your-own-data.md

```diff
@@ -77,6 +77,25 @@ wget -O <FILE_PATH_TO_SAVE_TO> <URL>
 *Specific example:* Here's an example where we will download that same array express file, but instead save it to the `data` folder and call it `some_array_data.zip`.
 (Best to keep the file extension consistent to avoid troubles!)
 
+Before we can `wget` the files, we should navigate to our home directory and make a `data/` folder to save to.
```

Comment:
I think this section should go before this step or before the previous step (which would be step 2). My reason for this is that participants, if like me, may try the commands as they go through the documents, which means if they do not already have a data folder to save to, they will save the file from the previous step directly in their home directory.

Reply:
Well, the first example doesn't use the `-O` option so the `data` folder isn't needed. The specific example this is under. I'll try to see if there's a way I can make this more clear.